src/extension/features/accounts/hide-memos/components/hide-memos-button.jsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import { controllerLookup } from 'toolkit/extension/utils/ember';
+import { l10n, getToolkitStorageKey } from 'toolkit/extension/utils/toolkit';
+
+export const HideMemosButton = ({ toggleHiddenState }) => {
+  const notHidden = getToolkitStorageKey('hide-memos', true);
+  const label = notHidden ? l10n('app.show', 'is-checked') : l10n('app.hide', 'not-checked');
+
+  const toggleHidden = () => {
+    toggleHiddenState(!notHidden);
+    controllerLookup('application').send('closeModal');
+    document.getElementById('ember1434').remove();
+  };
+
+  return (
+    <div
+      title="Hide Memo Column"
+      id="ember1434"
+      className="modal-account-view-menu ynab-checkbox ember-view"
+    >
+      <button
+        onClick={toggleHidden}
+        className={`ynab-checkbox-button js-ynab-checkbox-button ${label}`}
+        aria-label="Check Hide Memo Column"
+        role="checkbox"
+      >
+        <div className={`flaticon stroke ynab-checkbox-button-square ${label}`}></div>
+      </button>
+      <label className="label-checkbox">&nbsp;Hide Memo Column</label>
```

Comment:
Since we're injecting into the existing UI we should go with "Show Memo Column" to match their Running Balance option.

Also, this is where we should use localization:

```jsx
<label className="label-checkbox">{' '}{l10n('toolkit.hideMemoColumn', 'Show Memo Column')}</label>
```

the `{' '}` is syntactic sugar for your `&nbsp`

Reply:
Okay, i see what's going on here.  Does this mean we want the default state of the memo field to turn hidden on enable of feature, then they use view to disable/enable it quickly on the account page?  ahhh, smart.